corehq/apps/hqwebapp/templatetags/proptable_tags.py

```diff
@@ -20,7 +20,6 @@
 import pytz
 import six
 from jsonobject.exceptions import BadValueError
-from six.moves import map, zip_longest
```

Comment:
Does this need to be replaced with
```python
from itertools import zip_longest
```
? It's used on line 233/230, and I can't see where it's being imported.